src/gltf-component-mappings.js

```diff
@@ -121,23 +124,21 @@ AFRAME.GLTFModelPlus.registerComponent("waypoint", "waypoint", (el, componentNam
   el.setAttribute("waypoint", componentData);
 });
 
-AFRAME.GLTFModelPlus.registerComponent("media-frame", "media-frame", (el, componentName, componentData, components) => {
-  el.setAttribute("networked", {
-    template: "#interactable-media-frame",
-    owner: "scene",
-    persistent: true,
-    networkId: components.networked.id
-  });
-  el.setAttribute("shape-helper", {
-    type: "box",
-    fit: "manual",
-    halfExtents: {
-      x: componentData.bounds.x / 2,
-      y: componentData.bounds.y / 2,
-      z: componentData.bounds.z / 2
-    }
-  });
-  el.setAttribute("media-frame", componentData);
+import { findAncestorWithComponent } from "./utils/scene-graph";
+import { createElementEntity } from "./utils/jsx-entity";
+/** @jsx createElementEntity */ createElementEntity;
+
+AFRAME.GLTFModelPlus.registerComponent("media-frame", "media-frame", (el, _componentName, componentData) => {
```

Comment:
Ultimately GLTF loading will eventually go entirely through the new system of inflators, but for now just exposing media frames here is useful, even if slightly awkward.